test/cli/package-prefix-enforcement/test.out

```diff
@@ -8,20 +8,32 @@ nested/nested.rb:5: File belongs to package `Root::Nested` but defines a constan
 nested/nested.rb:40: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
     40 |  sig {returns(NilClass)}
           ^^^^^^^^^^^^^^^^^^^^^^^
-  Note:
-    Attempting to define class or method behavior in package namespace `Root`
+    nested/nested.rb:36: Attempting to define class or method behavior in `Root`
+    36 |module Root
+               ^^^^
+    __package.rb:3: Enclosing package declared here
+     3 |class Root < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^^^^
 
 nested/nested.rb:41: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
     41 |  def self.method
           ^^^^^^^^^^^^^^^
-  Note:
-    Attempting to define class or method behavior in package namespace `Root`
+    nested/nested.rb:36: Attempting to define class or method behavior in `Root`
+    36 |module Root
+               ^^^^
+    __package.rb:3: Enclosing package declared here
+     3 |class Root < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^^^^
 
 nested/nested.rb:37: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
     37 |  extend T::Sig
           ^^^^^^^^^^^^^
-  Note:
-    Attempting to define class or method behavior in package namespace `Root`
+    nested/nested.rb:36: Attempting to define class or method behavior in `Root`
+    36 |module Root
+               ^^^^
+    __package.rb:3: Enclosing package declared here
+     3 |class Root < PackageSpec
+        ^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
In my experience, packaging errors are the most confusing when I thought I was working in one package and suddenly I've opened up a file or folder that's actually in a nested package. For example `lib/my_app/` and `lib/my_app/data/`.

In times when this happens, it's usually getting to see that there are two `__package.rb` files in play that clear up the confusion. One suggestion might be:

```
nested/nested.rb:37: This file must only define behavior in enclosing package `Root::Nested` https://srb.help/3713
    37 |  extend T::Sig
          ^^^^^^^^^^^^^
  nested/nested.rb:36: Defining behavior in `Root` instead:
    36 |module Root
        ^^^^^^^^^^^
  nested/__package.rb:3: Enclosing package `Root::Nested` declared here
   3 |class Root::Nested < PackageSpec
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  __package.rb.rb:3: Package `Root` declared here
   3 |class Root < PackageSpec
      ^^^^^^^^^^^^^^^^^^^^^^^^
```

(not strongly attached to this)